guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -1,30 +1,56 @@
 [id="Installing_Puppet_Agent_during_Host_Provisioning_{context}"]
 = Installing Puppet Agent during Host Provisioning
 
-Install the Puppet agent during the host provisioning process.
+Use this procedure to install the Puppet agent during the host provisioning process.
+
+ifdef::orcharhino,satellite[]
```

Comment:
@maximiliankolb can you verify if orcharhino ships a vendored Puppet in their client repositories? Or it is like Katello and relies on upstream repositories?